### ISSUE TITLE
loading screen theme

### ISSUE THREAD

**Issue by DragonKEME** [RocketRobz/TWiLightMenu#374]
If the loading screen theme is dark, the other screen rest white
![img_20190203_160733](https://user-images.githubusercontent.com/40547592/52178413-80d99480-27ce-11e9-9acc-ff7bcb13570d.jpg)

**Comment by Volti-p** [comment]
Yes, because It's not implemented yet, that it Change both screens.
